modules/update-mirror-repository.adoc

```diff
@@ -114,6 +114,19 @@ $ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mi
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
 
+... Obtain the image signature for the version that you are upgrading to from either link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com] or link:https://storage.googleapis.com/openshift-release/official/signatures[Google Cloud Storage (GCS)].
+
+... Use `oc` command-line interface (CLI) to log in to the cluster that you are upgrading.
+
+... Apply the mirrored release image signature config map to the connected cluster:
++
+[source,terminal]
+----
+$ oc apply -f <image_signature_file> <1>
+----
++
+<1> For `<image_signature_file>`, specify the path and name of the file, for example, `mirror/config/signature-sha256-81154f5c03294534.yaml`.
```

Comment:
nit: probably `oc apply -f ${REMOVABLE_MEDIA_PATH}/mirror/config/<image_signature_file>` or some such above, and then the `<image_signature_file>` placeholder can just talk about the final filename like `signature-sha256-81154f5c03294534.yaml`.